### PR TITLE
Feature/remove organization

### LINKED ISSUE #338: Add delete functionality for removing organizations, departments and services
As a superuser I want to be able to delete organizations, departments or services so that I can remove obsolete or erroneous units.

## Definition of done
- [ ] A superuser can delete organizations, departments or services.
  - [ ]  If an organization is deleted, there will a cascading delete, where relevant departments, services and integrations (Github repos) are removed (departments belonging to that organization, services linked to departments that are being deleted, integrations linked to services to be removed).
    - [ ] The user gets a warning about deleting also N departments, N services and N integrations.
  - [ ] If a department is deleted, there will be a cascading delete for services and integrations (services linked to the department, integrations linked to the services to be deleted.)
    - [ ] The user gets a warning about deleting also N services and N integrations.
  - [ ] If a service is deleted, also relevant integration(s) are removed.

Note that 
 - [ ] Github repositories can potentially belong to more than one service. They can only be removed after checking that they are not linked to a service that remains on the portal. This applies to all cascading delete cases above.

### PR DESCRIPTION
**Pull request by elnzv**
Closes #338 

# Proposed changes:
- Added methods to delete organization, department(s), service(s)
- Buttons to delete
- Modals with confirmation message

### CODE REVIEW

**Comment by bajiat** [comment]
@elnzv Can you let us know about the status of the PR?

**Comment by elnzv** [comment]
@bajiat, have a button with modal now.
Working today on method to delete organization and its children.

**Comment by frenchbread** [comment]
@elnzv When you add "cleanup unneeded synced data" feature, please put it into a meteor method, so that we could reuse it later.

**Comment by elnzv** [comment]
@Digipalvelutehdas/dipor-team, please review.

File: departments/server/methods.js [hunk 1]
```diff
@@ -34,5 +34,45 @@ Meteor.methods({
...
+        console.log(error.message);
+      }
+    }
+  },
+  deleteDepartments (departments) {
+    departments.forEach(function(department){
+      Departments.remove(department);
+    });
+  },
+  "removeDepartmentMember" (departmentId, memberId) {
```
Comment (frenchbread) [inline departments/server/methods.js]: Remove quotes here.

File: departments/server/methods.js [hunk 2]
```diff
@@ -34,5 +34,45 @@ Meteor.methods({
...
+    // Get department
+    const department = Departments.findOne(departmentId);
+
+    // Update the department with new member IDs list
+    Departments.update(departmentId,{
+      $pull: {memberIds: memberId}
+    });
+  },
+  "removeDepartmentAdministrator" (departmentId, memberId) {
```
Comment (frenchbread) [inline departments/server/methods.js]: And here.

**Comment by frenchbread** [comment]
Tested, looks & works good. Merging..

**Pull request merged**